### ISSUE TITLE
Timer that triggers a macro doesn't work in version 3.5

### ISSUE THREAD

**Issue by JimmyKer** [reef-pi/reef-pi#1252]
**Describe the bug**
A timer that is set to execute a macro, doesn't work, the macro never gets started.

**To Reproduce**
Steps to reproduce the behaviour:
1. Create a new macro with one step: **alert**
2. Test the new created macro, this should send an **email alert**.
3. Create a **new timer** that execute the macro created in step 1.

**Expected behaviour**
I would expect the timer to trigger the macro and send an email alert.

**Desktop (please complete the following information):**
 - Pi Zero WH
 - reef-pi 3.5
 - Browser Edge (Phone + Desktop)

**Additional context**
Of course I don't want to use the Timer to send email alerts.
I want to use the Timer-Macro to make sure the ATO is enabled, by checking this every xx hours.
It happened already a couple times that I accidentally disabled the ATO with my fat fingers on the phone.

**Comment by CTYReef** [comment]
If you haven't looked at the closed issues for similar problems, there was one that started in May and closed 27 days ago reference macros not working off of timers.  The solution was that the macros have to be reversible to work off of a timer. 
If you already know that, I can't help further.  If you have not tried that, there are also other considerations for a multistep macro and I had to do the following to make mine work off of timers (Reef Pi versions 3.4 and 3.5, maybe older versions, too, but not sure): 1. Make the macros reversible, 2. Write the macro steps in reverse order (last step first), 3. Inverting the On/Off or true/false state of the individual steps (I was controlling equipment and if I wanted a switch to be turned on, I had to write it as turn off).   I recommend that you look at the thread for that closed issue, as one of the other participants posted a link for a very good description of the issue on another forum.  Note also that if you want to do a stand-alone or manual test of a macro written as described above for a timer, you must select the "Revert" option in Reef-Pi 3.5.
